Cargo.toml

```
@@ -93,6 +93,11 @@ exclude = [
     "tools/usb/control-test",
 ]
 
+[workspace.package]
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
```

```suggestion
version.workspace = "2.1.0"
authors.workspace = ["Tock Project Developers <tock-dev@googlegroups.com>"]
edition.workspace = "2021"
```

That is my understanding of how this should look based on https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table . Not sure what version we should put there

---

I was about to ask about this. It seems like _some_ file needed to actually have contents.

---

Ooops my find and replace was too good